### DISCUSSION TITLE
Customize the Panel to have a fixed position.

### DISCUSSION THREAD

**Discussion by parvathy-mpac** [bvaughn/react-resizable-panels#426] [Q&A] [answered]
Lets say we have a Panel Group with 2 Panels inside of it. Is there a way we can customize them such that one of the Panel's position can be made fixed with respect to the window and with a width that could be greater than the width of the Panel Group?

**Comment by bvaughn** [comment]
I'm not really sure what you mean, particularly this part:
> with a width that could be greater than the width of the Panel Group

Fixed pixel sizes aren't something this library supports though FWIW (see [the FAQs for more info](https://github.com/bvaughn/react-resizable-panels?tab=readme-ov-file#can-panel-sizes-be-specified-in-pixels))

**Reply by bvaughn** [reply, marked as answer]
FYI fixed-size panels are now supported:

- option 1: https://react-resizable-panels.vercel.app/examples/fixed-size-panels
- option 2: https://react-resizable-panels.vercel.app/examples/disabled-panels
